src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -2434,5 +2438,53 @@ class Program
     int [|a|];
 }");
         }
+
+        [WorkItem(40053, "https://github.com/dotnet/roslyn/issues/40053")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
+        public async Task TestEqualsNullableAnnotation()
```

Comment:
One case we need to make sure is tested is generating operators on value types, which should not annotated the same as reference types.